tests/test_quickbooks_automatic_fields.py

```diff
@@ -106,6 +104,11 @@ def test_run(self):
                 # Verify the sync meets or exceeds the default record count
                 self.assertLessEqual(expected_count, record_count)
 
+                if stream == 'budgets': # Skip the pagination assertion for this stream
```

Comment:
Is this here because you're getting some of the test working for this stream so you're not just skipping it at the top of the whole test?

Reply:
Is there a better way to generify it rather than hardcoding the stream here?

Reply:
Yes we are able to verify only automatic fields were sent to target, but not able to verify this for multiple pages of data.